Course_Part_04/src/Assingment12.java

```diff
@@ -3,6 +3,7 @@ public static void main(String[] args) {
         printGreetingMessageToConsole ();
```

Comment:
Prima uitwerking! Alleen een CTRL + ALT + L (code formatting) om het wat netter te maken. Haakjes in de return mag je weg laten.